app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: 'post_id'
+  has_many :likes, foreign_key: 'post_id'
+
+  belongs_to :user, foreign_key: 'author_id'
+
```

Comment:
- [ ] A post should "belongs to" an `author` because we already specified the column to be `author_id` in the `posts` table. Kindly modify the highlighted code as given below 👇
```ruby
belongs_to :author, class_name: 'User'
```

Reply:
Applied :) Thanks!